LIMS_IMAGE/web/Dockerfile

```diff
@@ -1,13 +1,16 @@
 FROM python:3.7
 ENV PYTHONBUFFEREED=1
 WORKDIR /src
+RUN apt-get update -y
+RUN apt-get install zbar-tools -y
 RUN apt-get update && apt-get install -y --no-install-recommends \
     unixodbc-dev \
     unixodbc \
     libpq-dev \
     curl
 ADD requirements.txt requirements.txt
 RUN pip install -r requirements.txt
+RUN pip install -U matplotlib
```

Comment:
Would this be able to go into requirements.txt?

Reply:
No :( because its -U and not -r it doesn't work. I did try it though

Reply:
Okay just checking. Should be all good then